### ISSUE TITLE
Illegal division by zero at multi-bleu.perl

### ISSUE THREAD

**Issue by PineLover** [microsoft/MASS#45]
![Uploading image.png…]()

hi we tried to pre-train the model by entering as below
but we got the above message.... anyone knows whats wrong? 
is max epoch too small?
we just wanted to check as fast as we can

python train.py                                      \
--exp_name unsupMT_enfr                              \
--data_path ./data/processed/en-fr/                  \
--lgs 'en-fr'                                        \
--mass_steps 'en,fr'                                 \
--encoder_only false                                 \
--emb_dim 1024                                       \
--n_layers 6                                         \
--n_heads 8                                          \
--dropout 0.1                                        \
--attention_dropout 0.1                              \
--gelu_activation true                               \
--tokens_per_batch 1000                              \
--optimizer adam_inverse_sqrt,beta1=0.9,beta2=0.98,lr=0.0001 \
--epoch_size 1000                                  \
--max_epoch 1                                      \
--eval_bleu true                                     \
--word_mass 0.5                                      \
--min_len 5                                          \

**Comment by PineLover** [comment]
![image](https://user-images.githubusercontent.com/7054970/62273630-f28e5480-b478-11e9-8d07-2e06bc155056.png)

**Comment by StillKeepTry** [comment]
@PineLover Maybe raise epoch size to have a try?
